domain/src/test/java/org/oppia/android/domain/classify/rules/fractioninput/FractionInputIsEquivalentToAndInSimplestFormRuleClassifierProviderTest.kt

```diff
@@ -32,7 +32,7 @@ class FractionInputIsEquivalentToAndInSimplestFormRuleClassifierProviderTest {
       isNegative = false,
       value = 2
     )
-  private val NEGATIVE_WHOLE_NUMBER_VALUE_TEST_2 =
+  private val NEGATIVE_INTEGER_NUMBER_VALUE_TEST_2 =
     InteractionObjectTestBuilder.createWholeNumber(
       isNegative = true,
```

Comment:
Also, please use a different method here to create Negative integers (you can use `createSignedInt()` method).

Reply:
Ok @prayutsu sir